buildSrc/src/main/java/AndroidComponents.kt

```diff
@@ -3,5 +3,5 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 object AndroidComponents {
-    const val VERSION = "53.0.20200804130112"
+    const val VERSION = "54.0.20200806165119"
```

Comment:
Let's wait to land this until the [AC update](https://github.com/mozilla-mobile/fenix/pull/13394) lands

Reply:
This PR is failing for the same reasons the AC one is. I'm trying to figure out what the cause of the test failures are.

Reply:
Once we figure out that out, we can use the latest version of the nightly build.

Reply:
👍 awesome